src/Utils.test.js

```diff
@@ -41,24 +41,24 @@ test('Test getWorldFromString', () => {
 });
 
 test('Test getStartingPositionFromString', () => {
-    // (1, 1)
-    expect(getStartingPositionFromString('aa', 16, 8, 2, 3)).toStrictEqual({ x: 1, y: 1 });
+    // (1, 1) facing east
+    expect(getStartingPositionFromString('aab', 16, 8, 2, 3, 2)).toStrictEqual({ x: 1, y: 1, direction: 2 });
```

Comment:
For these tests, please use a `defaultDirection` that is different from any of the other numbers being tested, to make sure that we are using the right parameter. Such as 7 (rather than 2).